python/cudf/cudf/_lib/pylibcudf/strings/__init__.pxd

```diff
@@ -1,3 +1,3 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 
-from . import case
+from . import case, find
```

Comment:
I missed this in the `case` PR: this line should be a `cimport` not an `import`. We always want the `__init__.pxd` files to expose the Cython APIs, not the Python APIs, if that makes sense.
```suggestion
from . cimport case, find
```